app/src/main/res/values-es/strings.xml

```diff
@@ -655,4 +655,5 @@
   <string name="achievements_of_user">Logros del usuario: %s</string>
   <string name="menu_view_user_page">Ver página de usuario</string>
   <string name="copyright_popup">Por favor, sólo cargue imágenes que haya tomado usted mismo. Personas que carguen imágenes con derechos de autor serán bloqueadas. Esto aplica para la versión beta también. Gracias por probar la aplicación!</string>
+  <string name="your_feedback">Your Feedback</string>
```

Comment:
You do not need to add in every language. Just do it in main string.xml which is English. These will be automatically updated.